### PR TITLE
Add localization for UI strings in Italian

### PR DESCRIPTION
**Pull request by stepankuzmin**
Hey 👋

This PR adds translations for UI strings we use in GL JS into Italian. We translated the UI strings automatically using the Google Translate API, and we need to verify that the translation is correct.

To help us verify the translations, you can open the `Files changed` tab on this page and compare the translations with the original file [`src/ui/default_locale.js`](https://github.com/mapbox/mapbox-gl-js/blob/main/src/ui/default_locale.js).

If the translation looks good, you can approve it using the `Review changes` button in the `Files changed` tab and selecting the `Approve` radio button when submitting the review.

To improve the machine translation, you can click the plus sign next to the line you want to enhance and select the suggestion button on the toolbar.

After making the suggestions, you can mark the review as finished using the `Review changes` button and selecting the `Request changes` radio button.

Thanks for helping 🙌

### CODE REVIEW

File: src/ui/locales/it.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "Map.Title": "Carta geografica",
+    "NavigationControl.ResetBearing": "Ripristina la direzione verso nord",
+    "NavigationControl.ZoomIn": "Ingrandire",
+    "NavigationControl.ZoomOut": "Rimpicciolisci",
+    "ScaleControl.Feet": "piedi",
+    "ScaleControl.Meters": "m",
+    "ScaleControl.Kilometers": "km",
+    "ScaleControl.Miles": "me",
+    "ScaleControl.NauticalMiles": "nm",
+    "ScrollZoomBlocker.CtrlMessage": "Usa ctrl + scorri per ingrandire la mappa",
```
Comment (ae-mo) [inline src/ui/locales/it.js:20]: Not sure what keyboard/mosue control is "Scroll" referring to?

**Review by ae-mo — CHANGES_REQUESTED** [review]

File: src/ui/locales/it.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Attiva/disattiva attribuzione",
+    "AttributionControl.MapFeedback": "Feedback sulla mappa",
+    "FullscreenControl.Enter": "Entra a schermo intero",
+    "FullscreenControl.Exit": "Esci dalla modalità schermo intero",
+    "GeolocateControl.FindMyLocation": "Trova la mia posizione",
+    "GeolocateControl.LocationNotAvailable": "Posizione non disponibile",
+    "LogoControl.Title": "Logo della scatola delle mappe",
+    "Map.Title": "Carta geografica",
```
[the remark this answers is not in the document]
Comment (ae-mo) [reply src/ui/locales/it.js:11]: If "geographic" has to be there, then this is fine.

File: src/ui/locales/it.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "NavigationControl.ResetBearing": "Ripristina la direzione verso nord",
+    "NavigationControl.ZoomIn": "Ingrandire",
+    "NavigationControl.ZoomOut": "Rimpicciolisci",
+    "ScaleControl.Feet": "piedi",
+    "ScaleControl.Meters": "m",
+    "ScaleControl.Kilometers": "km",
+    "ScaleControl.Miles": "me",
+    "ScaleControl.NauticalMiles": "nm",
```
[the remark this answers is not in the document]
Comment (stepankuzmin) [reply src/ui/locales/it.js:19]: Yep, this is a US version of the nautical mile :)

File: src/ui/locales/it.js (hunk 1 above)
Comment (stepankuzmin) [reply src/ui/locales/it.js:20]: This is a text on map blocker alert when cooperative gestures are enabled, see https://docs.mapbox.com/mapbox-gl-js/example/cooperative-gestures/

Screenshot 2022-04-12 at 13 08 03

Comment (ae-mo) [reply src/ui/locales/it.js:20]: I checked [apple support in Italian for multi touch](https://support.apple.com/it-it/HT204895), `scorri` should be correct.

Comment (ae-mo) [reply src/ui/locales/it.js:20]: But also in general it is a good translation 👌

Comment (stepankuzmin) [reply src/ui/locales/it.js:20]: @ae-mo, could you please approve this PR if everything looks good?

**Review by ae-mo — APPROVED** [review]

**Pull request merged**